### ISSUE TITLE
Write the Docs conference travel

### ISSUE THREAD

**Issue by kelle** [astropy/astropy-project#62]
The CoCo and interim finance committee have approved the travel of @jonathansick and @lglattly to attend the [Write The Docs conference](https://www.writethedocs.org/conf/portland/2020/) with an approximate budget of $1500 each, coming from the Moore grant. The purpose of this issue is to coordinate that travel and reimbursement while lending as much to transparency as possible. 

JSick and Lauren: Please move forward with planning your travel! Please keep receipts and what not. At the end of the day, you'll be submitting a reimbursement request to NumFocus. I'll send you our travel policy and reimbursement policy as soon as we have it finalized. I think the most important points at this juncture is that you should book economy but since this is coming from a private foundation, it doesn't need to be US Flag carrier.  

If you prefer I use the project PEX card to purchase flight and/or registration, please let me know and I'm happy to do that.

**Comment by jonathansick** [comment]
Thank you so much for this opportunity! We look forward to bringing some lots of ideas back to the Astropy project to improve our docs and user experience!

**Comment by jonathansick** [comment]
@lglattly I suggest that we plan to participate in the Sunday May 3 [writing day](https://www.writethedocs.org/conf/portland/2020/schedule/#sunday-may-3) so we can work on some tangible Astropy docs and maybe even rope in some other attendees :)

**Comment by lglattly** [comment]
Thanks for coordinating this @kelle! This is a great opportunity and will be very useful to our work with the Astropy Project! @jonathansick I'm on board to attend the May 3 writing day! I'll start looking into registering for the conference and booking flights and travel. Thanks again!

**Comment by lglattly** [comment]
@kelle when we purchase our [conference tickets](https://www.writethedocs.org/conf/portland/2020/tickets/), do we purchase corporate tickets for $500 or independent tickets for $275? Do we have non-profit status through NumFocus?

**Comment by kelle** [comment]
$275 non-profit

**Comment by pllim** [comment]
BTW, some conference (unrelated to Astropy stuff) is delayed due to COVID-19. You might want to think about the possibility of this (and any other) conference being affected and how this would in turn affect the flow of money.

For example, cancelling a plane ticket would incur extra charges, and so on.

FYI.

**Comment by lglattly** [comment]
Good point @pllim. @kelle do you think we should purchase travel insurance when we buy our plane tickets? I'm guessing if they canceled the conference due to COVID-19 they would refund our tickets, but I can double check the policy before buying a conference ticket.

**Comment by kelle** [comment]
Totally worth looking into travelers insurance. Please double check that anything COVID-19 related will be a covered incident.

**Comment by lglattly** [comment]
Write the Docs released a [COVID-19 update](https://www.writethedocs.org/conf/portland/2020/news/covid-19-update-01/) on their website today that says as of today the conference is scheduled to go ahead but they are keeping a close eye on the situation and will provide regular updates. It also says if a registrant is sick at the time of the conference, they should not attend and will be fully refunded. I am emailing them to double check that we would also be refunded in the event that they cancel the conference entirely. 

I looked into travelers insurance with United and it only covers your ticket if you cannot travel because you are sick with COVID-19, not if an event is canceled because of COVID-19. Other airlines might be different but my feeling is the travelers insurance isn't worth it. But, United is currently waiving cancellation fees on tickets purchased between now and March 31, so if you buy a flexible economy ticket you can cancel without being charged and get a full refund, but those flexible tickets are about $100 more than regular economy tickets, so not sure if they will work with the budget.

As for hotels, I looked at the policies of a couple that the conference recommends and they all say you just need to cancel more than 48 hours in advance to not be charged.

So what do you think @kelle, should @jonathansick and I still move ahead with registering for the conference at this point?

**Comment by lglattly** [comment]
Actually, a quick revision to the above. The hotels near the conference are all $160-$200/night (non-refundable) or a bit more for refundable rooms, so it will be hard to stay within the budget with a refundable room. Let me know if that changes anything!

**Comment by jonathansick** [comment]
https://www.writethedocs.org/conf/portland/2020/news/covid-19-update-02/

WTD just announced they're postponing the May dates and they're hoping to announce new dates by March 20 — likely looking at August.

I haven't bought my travel yet, but it sounds like I'm safe to buy a conference ticket.

**Comment by jonathansick** [comment]
Correction: they're not selling new tickets until they confirm the new dates. So I'll be on the lookout to buy once things settle down.

**Comment by lglattly** [comment]
Thanks for the update @jonathansick! I also haven't purchased a conference ticket or booked any travel or accommodations yet, was waiting to hear what they would announce. I will look into this again once they have the new dates and schedule.

**Comment by kelle** [comment]
Looks like this has gone virtual! https://www.writethedocs.org/conf/portland/2020/news/online-ticket-sales/

@lglattly , @jonathansick , Let us know if you're still planning to attend. We have a purchasing card which we could use to buy the tickets.

**Comment by lglattly** [comment]
I'm still very interested in attending! I think I could still get a lot out of this virtually. Thanks for the update @kelle!

**Comment by kelle** [comment]
@lglattly Should we go ahead and buy the ticket for you? I think that would be easier than dealing with reimbursement.

**Comment by lglattly** [comment]
@kelle Yes that would be great if that works for you! Just let me know if there is anything I need to do at this point. Thank you!

**Comment by kelle** [comment]
Ticket purchased. Lauren should get an email.

**Comment by lglattly** [comment]
Got it, thanks!

**Comment by hamogu** [comment]
@lglattly @jonathansick Did both of you attend the conference?

Please provide a short summary (2-3 sentences) on your experience, work for astropy, and lessons learned.
If you gave a talk, presented work etc., please provide a link or copy and paste the abstract here.

This is both to keep a record for the future and make sure that the things you learned profit the entire project, but also to help us write the annual report for the Moore grant that funded this.

**Comment by lglattly** [comment]
@hamogu I attended Write the Docs virtually from Sunday, August 9, 2020 to Tuesday, August 11, 2020. All of the talks at the conference centered on producing technical documentation for a variety of open source projects.

I learned the most from the talks focusing on style guides and language editing, as that is my area of expertise. Through the talks and the materials shared during and after, I was introduced and given access to several style guides and other writing resources I was not aware of previously. I found the talk on [accessibility in documentation](https://www.youtube.com/watch?v=PS8ywGg0x-U&list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ&index=2) to be the most useful, and I now reference the presenter's [Accessibility style guide](https://github.com/heyawhite/tech-writing-tools/blob/main/accessibility/style.md) during my copy editing work for Astropy.

The information shared at the Write the Docs conference gave me ideas of additions and revisions I would like to make to the [Astropy Narrative Style Guide](https://docs.astropy.org/en/latest/development/style-guide.html) at some point in the future, and I appreciated the opportunity to connect with other writers and editors working in the world of technical documentation!
